### PR TITLE
Language improvements.

### PR DESCRIPTION
**Pull request by sara-02**
1. Enhance the description of Pycon Pune and Dev Sprint.
2. Hyperlink -Department of Computer Engineering and Information Technology, COEP Pune.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -32,10 +32,10 @@ <h2 class="fs-2">College of Engineering, Pune, India</h2>
     <div class="responsive-container">
         <h2>About</h2>
         <p>
-          PyCon,  the gathering for the community using and developing the open-source
-          Python programming language. This is the first year of the PyCon Pune where
```
Comment (sayanchowdhury) [inline templates/index.html:36]: We picked this definition from PyCon US

File: templates/index.html [hunk 2]
```diff
@@ -32,10 +32,10 @@ <h2 class="fs-2">College of Engineering, Pune, India</h2>
...
-          the community will meet for two days of talks and working on upstream
-          projects in two days of dev sprint.
+         PyCon is a gathering of the community using and developing Python;
+         the open-source programming language.
+         This is the first year of the PyCon Pune where the community will meet for two days of talks,
+         and for working on upstream projects during two days of the dev sprint.
```
Comment (sayanchowdhury) [inline templates/index.html:37]: +1 to this

File: templates/index.html [hunk 3]
```diff
@@ -108,7 +108,9 @@ <h2 class="fs-2">College of Engineering, Pune, India</h2>
           <div class="skew-down"></div>
           <div class="circle">Sprints</div>
           <div class="b-f content">
-            Development sprints are a key part of PyCon, and a chance for the contributors to open-source projects to get together face-to-face for up to two days of intensive learning, development and camaraderie.
+            Development sprints are a key part of PyCon.
+            A chance for the open-source contributors, evangelists, and newbies
+            to meet on-site for up to two days of intensive learning, development, and camaraderie.
```
Comment (sayanchowdhury) [inline templates/index.html]: This is also picked from the PyCon US

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 3 above)
Comment (sara-02) [reply templates/index.html]: I am trying to include newbies in the dev sprints.
"Development sprints are a key part of PyCon, and a chance for the contributors to open-source projects to get together face-to-face for up to four days of intensive learning, development and camaraderie. Newbies sit with gurus, go out for lunch and dinner together, and have a great time while advancing their project."- This is the complete description of what Pycon US says. We need to find a way to include a sentence about new contributors as well :)

File: templates/index.html (hunk 1 above)
Comment (sara-02) [reply templates/index.html:36]: Reckon that, will revert the changes.

File: templates/index.html (hunk 3 above)
Comment (sayanchowdhury) [reply templates/index.html]: In that case just change `PyCon. A` to `PyCon, a`

**Comment by sara-02** [comment]
Added the changes. Added a new line for new contributors, hope it goes well.

**Pull request closed without merging**